[plugins.subscribe] ignore the subscribe commands in the subscribe-handlers

deffnull opened the pull request:
- allows to configure global_subscribes without immediately triggering them

deffnull committed:
[plugins.subscribe] ignore private chats with the bot

- allows to configure global_subscribes without immediately triggering them

das7pad commented:
Actually I do like the ability to trigger Hangouts via this plugin from the private chat with the bot. But I do agree that `/bot global_subscibe keyword` should not trigger a notification for `keyword` in another chat.

What do you think about a context-flag that disables the (global-)subscribe handler in case the user has changed (global-)subscribes? e.g. `hide_from_subscribes`

das7pad committed:
[plugins.subscribe] ignore the module commands in the subscribe handlers

keep the idea of 6a75f68f1c42f4883f94ae979a6ccc0c7620b4e3 but do not ignore all private chat messages

das7pad committed:
[plugins.subscribe] allow other modules to hide their commands

das7pad commented:
As the command is executed after the event has reached the subscribe handlers, a slightly different approach was needed:
https://github.com/das7pad/hangoutsbot/compare/v3.1-das7pad...plugins/subscribe

https://github.com/das7pad/hangoutsbot/commit/f24a6db08456b58f2befc3b2043d298f70cb56ec also adds a shared to hide other commands from subscribes. I would add `tldr` and the remind commands to the ignored commands as well.

das7pad committed:
Merge branch 'v3.1-das7pad' into feature/ignore-pvt-botchats

das7pad commented:
Thanks!
